source/bson-binary-vector/tests/README.md

```diff
@@ -0,0 +1,40 @@
+# Testing Binary subtype 9: Vector
+
+The JSON files in this directory tree are platform-independent tests that drivers can use to prove their conformance to
+the specification.
+
+These tests focus on the roundtrip of the list numbers as input/output, along with their data type and byte padding.
+
+Additional tests exist in `bson_corpus/tests/binary.json` but do not sufficiently test the end-to-end process of Vector
+to BSON. For this reason, drivers must create a bespoke test runner for the vector subtype.
+
+Each test case here pertains to a single vector. The inputs required to create the Binary BSON object are defined, and
+when valid, the Canonical BSON and Extended JSON representations are included for comparison.
+
+## Version
+
+Files in the "specifications" repository have no version scheme. They are not tied to a MongoDB server version.
+
+## Format
+
+#### Top level keys
+
+Each JSON file contains three top-level keys.
+
+- `description`: human-readable description of what is in the file
+- `test_key`: Field name used when decoding/encoding a BSON document containing the single BSON Binary for the test
+  case. Applies to *every* case.
+- `tests`: array of test case objects, each of which have the following keys. Valid cases will also contain additional
+  binary and json encoding values.
+
+#### Keys of tests objects
```

Comment:
This file should describe all the assertions that the test runner should make.  Many are obvious, but one that might be missed, for example, is that you can round trip from vector to bson and then back to vector.

IIRC the bson_corpus README does this pretty well.

Reply:
I'm not sure if this is outdated now. If you need more, would you please be specific? I am also happy to revisit once another driver takes a stab at it.

Reply:
We discussed this in person, so I think we're in alignment now.  The idea is to just look at all the assertions being done in the Python driver test runner, and write them down in prose here.

Reply:
2d4ea724 - Added Required Tests section to README. Removed JSON from tests.